src/main/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -354,7 +357,7 @@ protected int doWork() {
                     rec.getReadName().equals(opticalDuplicateQueryName) ||
                     recordInFileIndex == nextOpticalDuplicateIndex;
 
-            rec.setAttribute(DUPLICATE_TYPE_TAG, null);
+            if (CLEAR_DT) rec.setAttribute(DUPLICATE_TYPE_TAG, null);
```

Comment:
I realize a lot of code here isn't written this way, but I think we should migrate to the Java coding standards in the picard code base. That means even a one-line `if` like this is written on two lines and has `{` `}` around its body.